python/paddle/fluid/layers/sequence_lod.py

```diff
@@ -1380,6 +1381,21 @@ def sequence_mask(x, maxlen=None, dtype='int64', name=None):
             #  [1 1 1 1 1 1 1 1 0 0]]
 
     """
+
+    if _non_static_mode():
+        if not isinstance(dtype, core.VarDesc.VarType):
+            dtype = convert_np_dtype_to_dtype_(dtype)
+        if in_dygraph_mode():
```

Comment:
why not set in_dygraph_mode check first to speed up eager call

Reply:
Thx, i will set in_dygraph_mode check first in next PR.